internal/fs/fs.go

```diff
@@ -270,9 +271,19 @@ func CopyDir(src, dst string) error {
 // the copied data is synced/flushed to stable storage.
 func copyFile(src, dst string) (err error) {
 	if sym, err := IsSymlink(src); err != nil {
-		return err
+		return errors.Wrap(err, "symlink check failed")
 	} else if sym {
-		return cloneSymlink(src, dst)
+		if err := cloneSymlink(src, dst); err != nil {
+			if runtime.GOOS == "windows" && strings.HasSuffix(err.Error(), "A required privilege is not held by the client.") {
```

Comment:
this isn't i18n-friendly. not a blocker, necessarily, but is there any way we could sniff the system actual error type here? obviously that means syscall, so this'd have to be a in a windows-tagged file.